lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -145,6 +146,13 @@ def _init_job_rec(self, user_id, params):
         with self.get_mongo_util().mongo_engine_connection():
             job.save()
 
+        create_job_status_msg = {
```

Comment:
I would have a field that goes in every message that is something like "event_type", which could be "created", "canceled", "queued", etc. That way consumers can switch on this field and know it'll always be there. It would also convey the meaning of each message/event.